.github/workflows/docs-ci.yml

```diff
@@ -7,22 +7,32 @@ on:
       - 0.8.x
 
 jobs:
-  upload_docs:
-    name: update docs
+  update_docs:
+    name: Update docs
     runs-on: ubuntu-latest
+
     env:
-      GRGIT_USER: ${{ secrets.GRGIT_USER }}
-      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+      GITHUB_TAG_NAME: ${{ github.event.release.tag_name }}
+      GITHUB_BRANCH_NAME: ${{ github.ref }}
```

Comment:
Providing these environment variables will show the version the docs were built from in the dokka header (they are queried [here](https://github.com/kordlib/kord/blob/b5d5dc82f85d145741d71629e179cba1125592db/buildSrc/src/main/kotlin/Projects.kt#L9)):

![dokka header](https://user-images.githubusercontent.com/47486203/185756191-8d57e1d0-cc56-4b82-97a1-1475f6f2bfe8.png)